CONTRIBUTING.md

```diff
@@ -67,6 +67,16 @@ For development you are going to want to checkout out the source. To get it, clo
     cd linguist/
     script/bootstrap
 
+To run Linguist from the cloned repository directory, you will need to generate the code samples first:
```

Comment:
**Nitpick:** *"cloned repository directory"* is a bit redundant. We can shorten that to simply "cloned repository" instead.